app/src/main/java/org/mozilla/fenix/compose/ThumbnailCard.kt

```diff
@@ -50,14 +54,16 @@ import org.mozilla.fenix.theme.Theme
 fun ThumbnailCard(
     url: String,
     key: String,
+    size: Dp = 108.dp,
+    backgroundColor: Color = colorResource(id = R.color.photonGrey20),
     modifier: Modifier = Modifier,
     contentDescription: String? = null,
     contentScale: ContentScale = ContentScale.FillWidth,
     alignment: Alignment = Alignment.TopCenter
 ) {
     Card(
         modifier = modifier,
-        backgroundColor = colorResource(id = R.color.photonGrey20)
+        backgroundColor = backgroundColor
```

Comment:
This could also be made into a follow investigation issue so we aren't blocking from landing, but I mostly want to verify what we are doing in the xml currently based on your current findings, and whether or not if we used layer2 if that would affect the other usages of `ThumbnailCard` such as in `RecentBookmarks` - this might involve a UX decision.

The main concern would be that the current usage of the default card background color `ThumbnailCard` outside of the tab trays could appear wrong for the various themes, and we just haven't seen it because the favicon might not be shown often.